docs/apis/avalanchego/apis/info.md

```diff
@@ -236,14 +238,15 @@ curl -X POST --data '{
 {
   "jsonrpc": "2.0",
   "result": {
-    "version": "avalanche/1.4.10",
+    "version": "avalanche/1.9.1",
     "databaseVersion": "v1.4.5",
-    "gitCommit": "a3930fe3fa115c018e71eb1e97ca8cec34db67f1",
+    "gitCommit": "79cd09ba728e1cecef40acd60702f0a2d41ea404",
     "vmVersions": {
-      "avm": "v1.4.10",
-      "evm": "v0.5.5-rc.1",
-      "platform": "v1.4.10"
-    }
+      "avm": "v1.9.1",
+      "evm": "v0.11.1",
+      "platform": "v1.9.1"
+    },
+    "rpcProtocolVersion": "18"
```

Comment:
I think this will actually show up between the `databaseVersion` and the `gitCommit`